config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -82,6 +82,34 @@ postsubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: post-kubernetes-bazel-build
+    branches:
+    - master
+    decorate: true
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+    extra_refs:
+    - base_ref: master
+      org: kubernetes
+      repo: test-infra
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190420-93fab49-master
+        command:
+        - bash
+        args:
+        - -c
+        - |
+                set -o errexit
+                gcloud auth activate-service-account --key-file=/etc/service-account/service-account.json
```

Comment:
I think this is already part of the runner.sh, ref https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/runner.sh#L90-L92

Let's try to avoid bashaml :-)

Reply:
I do not want to use https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/runner.sh

* It is too complicated, with unnecessary functionality for this use case
* this script does not live in the repo and therefore cannot be changed with kubernetes/kubernetes PR
* This script is only two commands (and I plan to move it into a script in k/k.

Alternatively I can make kubernetes_bazel.sh activate the service account when the env var is present?